lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,58 @@ module ActiveModel
   class Serializer
     class Adapter
       class Attributes < Adapter
+        attr_accessor :_nested_associations
```

Comment:
So, the title is `Support nested associations for json adapter` so, probably should go in the Json adapter?